var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -221,7 +221,7 @@ def initconfig_compiler_entries(self):
         # Default entries are already defined in CachedCMakePackage, inherit them:
         entries = super().initconfig_compiler_entries()
 
-        if spec.satisfies("+rocm"):
+        if spec.satisfies("+rocm ^blt@:0.6.99"):
```

Comment:
Can you also fix the others? Then LGTM

Reply:
Sorry I didn't check the codebase and just merged your suggestion as an opportunity to finally get those tests to run :)